zne/noise_amplification/folding_amplifier/glorious_folding_amplifier.py

```diff
@@ -13,14 +13,16 @@
 """Glorious DAG Folding Noise Amplification (Temporary)"""
 import copy
 
-from qiskit.circuit.library import Barrier
 from qiskit.converters import circuit_to_dag, dag_to_circuit
 from qiskit.dagcircuit import DAGCircuit
 
 
 def _inverse_dag(dag_to_inverse: DAGCircuit) -> DAGCircuit:
```

Comment:
A good naming convention is:
1. Verbs for functions/methods
2. Nouns for variables/attributes/fields

I would therefore suggest renaming the function to something like `_invert_dag` 🙂